stores/common_test.go

```diff
@@ -686,6 +686,9 @@ func testBasicSubStore(t *testing.T, s Store) {
 }
 
 func testGetSeqFromStartTime(t *testing.T, s Store) {
+	limits := testDefaultStoreLimits
+	limits.MaxAge = 500 * time.Millisecond
```

Comment:
in the sake of test timing, any reason not to set this *really* low? like 10millis?